internal/handlers/common/aggregations/stages/projection/projection.go

```diff
@@ -150,13 +147,20 @@ func ProjectDocument(doc, projection *types.Document, inclusion bool) (*types.Do
 
 		switch idValue := idValue.(type) {
 		case *types.Document: // field: { $elemMatch: { field2: value }}
-			// TODO: https://github.com/FerretDB/FerretDB/issues/2633
-			return nil, commonerrors.NewCommandErrorMsg(
-				commonerrors.ErrCommandNotFound,
-				fmt.Sprintf("projection %s is not supported",
-					types.FormatAnyValue(idValue),
-				),
-			)
+			var op operators.Operator
```

Comment:
Also could you keep todo for testing https://github.com/FerretDB/FerretDB/issues/2678

Don't forget manual testing :wink:

Reply:
The TODO is in `type.go`. NotImplemented error is returned by `NewOperator`.